debian/control

```diff
@@ -0,0 +1,29 @@
+Source: libauxv
+Section: libs
+Priority: optional
+Maintainer: Frédéric Bonnard <frediz@linux.vnet.ibm.com>
+Standards-Version: 3.9.6
+Build-Depends: debhelper (>= 9),
+               dh-autoreconf
+Homepage: https://github.com/Libauxv/libauxv
+
+Package: libauxv1
```

Comment:
Actually, no. That's surprising at first :) but Debian Policy on library package naming is the following : 
"Normally, the run-time shared library and its SONAME symlink should be placed in a package named librarynamesoversion, where soversion is the version number in the SONAME of the shared library. "
in "8.1 Run-time shared libraries" from https://www.debian.org/doc/debian-policy/ch-sharedlibs.html